src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -2,7 +2,7 @@
 
 # AudioInputProvider
 
-The `AudioInputProvider` provides a list of the user's available `audio-input` devices and the currently selected device.
+The `AudioInputProvider` provides a list of the user's available `audio-input` devices, the currently selected device, and the error when selecting device failed.
```

Comment:
"when selecting device failed"  -> "if device selection fails"?

Reply:
Yes, this is much more clear! Will do!